presto-docs/src/main/sphinx/migration/from-hive.rst

```diff
@@ -132,3 +132,15 @@ Presto query::
 
     date_diff('day', startdate, enddate)
 
+Overwriting data on insert
+--------------------------
+
+By default, ``INSERT`` queries are not allowed to overwrite existing data. You
+can use the catalog session property ``insert_existing_partitions_behavior`` to
+allow overwrites. Prepend the name of the catalog using the Hive connector,
```

Comment:
There's another mode that's allowed: `APPEND`. We may want to document that, too, here or as a separate change.

Reply:
Also, I think that the `overwrite` mode support is not available for s3 backed tables - since there is no concept of stage-and-move to target directory. We could also add that in the documentation.

Reply:
I would like to link to the docs for that session property but still looking for a place to put that and then link to it. Do you think I should document it all here (ERROR as default, APPEND..) ?

Reply:
Good to know .. I can add all that .. but probably not here in the migration docs but somewhere in the hive connector docs ... but where? Should I start a separate section about inserting data ? Or session properties?

Reply:
Yeah, it makes sense to add it to the Hive connector docs. We don't have a section with the list of session properties, so maybe a dedicate section about insert behavior would be way to go. @electrum, do you have any thoughts?

Reply:
If I add it insert section to the hive docs the migration section can be shorter and mostly point there.

Reply:
Can you please chime in here @electrum to decide what direction we should take this.

Reply:
equivalent to hive insert overwrite

Reply:
We should mention this is equivalent to Hive's `INSERT OVERWRITE` syntax (documented at https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DML)

Reply:
Added as agreed.